torch/_tensor_docs.py

```diff
@@ -4635,3 +4635,22 @@ def callable(a, b) -> number
 in the output mirror changes in ``self``, and the output stays attached
 to the autograd graph. ``cls`` must be a subclass of ``Tensor``.
 """)
+
+add_docstr_all('pointers',
+               r"""
+pointers() -> IntTensor
+
+Returns the pointers of a GCS sparse tensor. These pointers represent
+the flattened row pointers of the N-D tensor that has been flattened
+using reduction as described in 
+https://github.com/Quansight-Labs/rfcs/tree/pearu/rfc0005/RFC0003-sparse-roadmap.
+""")
+
+add_docstr_all('reduction',
+               r"""
+reduction() -> Tensor
+
+Tensor which shows the demarcation between dimensions that will be
+flattened from the N-D tensor.
```

Comment:
This isn't enough description.

Reply:
For example, what is the dimensionality/size of this tensor, depending on the size of the sparse tensor? How exactly is the "Demarcation" represented? An example?